scripts/configurator.sh

```diff
@@ -1,11 +1,11 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
`if [[ ... ]]` syntax is not supported by `sh` (sh on OS X is just bash but it's actually sh on Debian & Ubuntu).